configs/config-no-entropy.h

```diff
@@ -0,0 +1,699 @@
+/**
+ *  Minimal configuration that does not require implementation of an entropy
```

Comment:
Can you reword to something like 'Minimal configuration of features that do not require an entropy source'.

Having 'implementation of an entropy source' doesn't sound right given the entropy source itself could be a hardware or some function of the unpredictable behaviour of the system.